editor/animation/animation_player_editor_plugin.cpp

```diff
@@ -2291,7 +2291,7 @@ AnimationPlayerEditor::~AnimationPlayerEditor() {
 void AnimationPlayerEditorPlugin::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_ENTER_TREE: {
-			Node3DEditor::get_singleton()->connect(SNAME("transform_key_request"), callable_mp(this, &AnimationPlayerEditorPlugin::_transform_key_request));
+			Node3DEditor::get_singleton()->connect(SNAME("transform_3d_key_request"), callable_mp(this, &AnimationPlayerEditorPlugin::_transform_3d_key_request));
```

Comment:
There were a few APIs in the animation editor, and this signal for communicating with `Node3DEditor`, with "transform" in the name, but these APIs only operate on 3D transform. I've renamed them with "3d" for clarity. These are internal APIs, not exposed anywhere.